### ISSUE TITLE
Quick overview opening slowly on larger presentations

### ISSUE THREAD

**Issue by jakobkordez** [slidevjs/slidev#1889]
Quick overview takes a few seconds to open on large presentations (400 slides).
This is a big problem when I want to quickly jump to a slide.

This problem occurs because a very large amount of DOM elements need to be added during opening.

This problem was fixed in #1610 but was never merged.

Is there a possibility of this being merged if I resolve the conflicts or should I just create a fork for my personal use?

**Comment by jakobkordez** [comment]
It does not work. I added the `overviewSnapshots: true` to the frontmatter but opening the quick overview takes a few seconds then my console starts spamming `[Slidev] Snapshot captured for slide xxx` and a bunch of `[Vue warn]` warnings. All of the slides get replaced with the symbols for missing images.
Even after all the slides have been replaced with missing images, opening and closing the quick overview takes a second, compared to the instantaneous opening and closing in my implementation.
Running `build` and then serving `dist/` does not make a difference.
